### ISSUE TITLE
Feature request: status line text with segment IDs

### ISSUE THREAD

**Issue by bertsky** [qurator-spk/dinglehopper#9]
To make navigation in the source annotation easier, show the current `TextRegion` / `TextLine`  / `Word` / `Glyph` ids in the browser's status bar.

**Comment by mikegerber** [comment]
Yes, that would be useful. This feature is somewhat connected to #10 as I need to do text extraction differently to retain the segment ids.

**Comment by bertsky** [comment]
Or you wait for the PAGE-XML DOM to give us references upwards in the hierarchy. See OCR-D/core#313 and discussion there.

**Comment by mikegerber** [comment]
Latest master now has a tooltip to display the segment id:

![image](https://user-images.githubusercontent.com/34309482/95453909-f9adce00-096b-11eb-9303-bd8c57d0a74f.png)

This is currently the region id for PAGE, and the texline id for ALTO as these are the levels we are currently extracting from. (Upcoming feature #5 will give more options.)

Support to display this for the word differences is also upcoming, until then this issue should stay open.
